test/helpers/reactjs_helper_test.rb

```diff
@@ -10,13 +10,17 @@ def webpack_asset_paths(bundle_name, opts)
   setup do
     Foreman::Plugin.register(:foreman_react) {}
     Foreman::Plugin.register(:foreman_angular) {}
-    @plugins = [Foreman::Plugin.find(:foreman_react), Foreman::Plugin.find(:foreman_angular)]
-    self.stubs(:all_webpacked_plugins).returns(@plugins)
+    Foreman::Plugin.any_instance.stubs(:uses_webpack?).returns(true)
```

Comment:
I'm a bit unfamiliar with stubbing. Is this stubbing removed on teardown?

Reply:
yes.